pkg/options/resource-provider.go

```diff
@@ -18,6 +19,13 @@ func NewResourceProviderOptions() resourceprovider.ResourceProviderOptions {
 		Pow:       GetDefaultResourceProviderPowOptions(),
 		IPFS:      GetDefaultIPFSOptions(),
 		Telemetry: GetDefaultTelemetryOptions(),
+		Preflight: preflight.PreflightConfig{
+			GPU: struct {
+				MinMemoryGB int64
+			}{
+				MinMemoryGB: preflight.RequiredGPUMemoryGB,
+			},
+		},
```

Comment:
Move this to the preflight package. We add items to the options package when they are configurable by users.